docs/schema/deploymentlookuplistmap-schema.md

```diff
@@ -34,23 +35,23 @@ The following table lists elements in the DeploymentLookupListMap.xsd schema fil
 </thead>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="lookupitem-element-deploymentlookuplistmap.md">LookupItem element (DeploymentLookupListMap)</a></span></p></td>
+<td align="left"><p><span><a href="lookupitem-element-deploymentlookuplistmap.md">LookupItem element (DeploymentLookupListMap)</a></span></p></td>
```

Comment:
No need to correct this issue, but you could also delete the entire HTML tag for "span."

An alternative would to line 38 would be:
`<td align="left"><p><a href="lookupitem-element-deploymentlookuplistmap.md">LookupItem element (DeploymentLookupListMap)</a></span></p></td>`